internal/workload/manager.go

```diff
@@ -220,6 +216,7 @@ func (w *WorkloadManager) Update(configuration models.DeviceConfigurationMessage
 func (w *WorkloadManager) ensureWorkloadDirExists(workloadName string) error {
 	workloadDirPath := w.getWorkloadDirPath(workloadName)
 	if _, err := os.Stat(workloadDirPath); err != nil {
+		/* #nosec */
 		if err := os.MkdirAll(workloadDirPath, 0755); err != nil {
```

Comment:
this 755 is ok?

Reply:
yes, is for the folder, no for the workload definition at all.